ubuntu-1804-image_5b461f730428630abc0bf442.md

```diff
@@ -213,14 +212,7 @@ Available versions:
 
 ### Rust
 
-Rust 1.33 is available via rustup, with appropriate language tools including:
-
-- cargo
-- rustc
-- rustdoc
-- rust-gdb
-- rust-lldb
-- rustup
+Available via docker image
```

Comment:
This is not helpful to end users. Let's just remove Haskell and Rust sections while we don't have any examples.

Reply:
Rust and haskell removed from file